bodhi/server/models.py

```diff
@@ -2451,22 +2451,31 @@ def edit(cls, request, data):
 
         up.date_modified = datetime.utcnow()
 
-        # Store the update alias so Celery doesn't have to emit SQL
-        update_alias = up.alias
-
         notifications.publish(update_schemas.UpdateEditV1.from_dict(
             message={'update': up, 'agent': request.user.name, 'new_bugs': new_bugs}))
 
-        # Commit the changes in the db before calling a celery task.
+        # If editing a Pending update, all of whose builds are signed, for a release
+        # which isn't composed by Bodhi (i.e. Rawhide), move it directly to Testing.
+        if not up.release.composed_by_bodhi and up.status == UpdateStatus.pending \
+                and up.signed:
+            log.info("Every build in the update is signed, set status to testing")
+            up.status = UpdateStatus.testing
+            up.date_testing = func.current_timestamp()
+            up.request = None
+            log.info(f"Update status of {up.alias} has been set to testing")
+
+        if config['test_gating.required']:
+            log.info(f"Updating test gating status of {up.alias}")
+            up.update_test_gating_status()
+
+        # Commit the changes in the db before calling celery tasks.
         db.commit()
 
-        handle_update.delay(
-            api_version=2, action='edit',
-            update_alias=update_alias,
-            agent=request.user.name,
-            new_bugs=new_bugs
-        )
+        log.info("Deferring working on bugs and fetching test cases to celery")
+        work_on_bugs.delay(up.alias, new_bugs)
```

Comment:
We had some weird bugs with celery serializing the objects when passing directly up.alias to the tasks. I ll recommend doing something like 
```suggestion
       update_alias = up.alias
        work_on_bugs.delay(update_alias, new_bugs)
```